src/main/row.ts

```diff
@@ -517,7 +517,7 @@ function filterMinistryAttendees(ministryNames: string[], ministryFilters: strin
  */
 function filterDomainAttendees(domainNames: string[], domainFilters: string[]): string[] {
     if (domainNames[0] === "Rotation") {
-        return getRotationMembers(); // TODO: Josh will implement this.
```

Comment:
This was throwing a compiler error because the function didn't exist causing the tests to freak out